pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -291,6 +290,10 @@ func (b *awsElasticBlockStoreBuilder) SetUpAt(dir string) error {
 		return err
 	}
 
+	if !b.readOnly {
```

Comment:
We need to ensure the post-condition.  If it is safe to assume that kubelet's fsgid is 0, then we can elide this call - is that a safe assumption?  I am not sure.